tests/marker_test.go

```diff
@@ -15,32 +15,39 @@
 package tests_test
 
 import (
+	"crypto/rand"
 	"fmt"
 	"time"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 
 	"github.com/kubevirt/bridge-marker/tests"
+	core_api "k8s.io/api/core/v1"
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 var _ = Describe("bridge-marker", func() {
 	Describe("bridge resource reporting", func() {
 		It("should be reported only when available on node", func() {
 
-			out, err := tests.RunOnNode("node01", "sudo ip link add br-test type bridge")
+			brId := make([]byte, 6)
+			rand.Read(brId)
+			uniqueBridgeName := fmt.Sprintf("br_%x", brId)
+			resourceName := core_api.ResourceName(fmt.Sprintf("%s/%s", "bridge.network.kubevirt.io", uniqueBridgeName))
+
+			out, err := tests.RunOnNode("node01", fmt.Sprintf("sudo ip link add %s type bridge", uniqueBridgeName))
 			if err != nil {
 				panic(fmt.Errorf("%v: %s", err, out))
 			}
 
-			defer tests.RunOnNode("node01", "sudo ip link del br-test")
+			defer tests.RunOnNode("node01", fmt.Sprintf("sudo ip link del %s", uniqueBridgeName))
```

Comment:
I think we can remove the defer here no?

Reply:
Done